src/select/__tests__/select.test.js

```diff
@@ -406,48 +372,31 @@ describe('Stateless select', function() {
           {isDropDownOpen: mode === TYPE.select},
         ],
         [KEY_STRINGS.Escape, 'close dropdown', {isDropDownOpen: false}],
-        [
```

Comment:
why is this test moved to another place. We re-used more code in here and it was still readable

Reply:
backspace no longer changes internal component state, it emits an `onChange` event, and all of these test were just asserting on component state and not checking for calls to `onChange`. keeping the backspace test here would have resulted in a few more conditional statements that make this code more convoluted.